lib/bundle/locker.rb

```diff
@@ -0,0 +1,100 @@
+# frozen_string_literal: true
+
+module Bundle
+  module Locker
+    module_function
+
+    def lock(entries)
+      lockfile = Brewfile.path.dirname/"Brewfile.lock.json"
+
+      lock = if lockfile.exist?
+        JSON.parse(lockfile.read)
```

Comment:
How about a single-line `lock = JSON.parse(lockfile.read) if lockfile.exist?`?

Reply:
I think that made more sense before refactoring but: yes, why not.